uuidv8.go

```diff
@@ -258,3 +260,32 @@ func (u *UUIDv8) UnmarshalJSON(data []byte) error {
 	*u = *parsed
 	return nil
 }
+
+// Value implements the driver.Value interface for database writes.
+func (u *UUIDv8) Value() (driver.Value, error) {
+	if u == nil || len(u.Node) != 6 {
```

Comment:
For me the len(u.Node) != 6 should return an error, no?

But maybe I'm wrong

Reply:
Makes sense 💯 I've made the validation stricter here.